src/renderer/components/ft-video-player/ft-video-player.js

```diff
@@ -966,6 +972,45 @@ export default Vue.extend({
       videojs.registerComponent('fullWindowButton', fullWindowButton)
     },
 
+    createTheatreModeButton: function() {
+      if (!this.$parent.theatrePossible) {
+        return
+      }
+
+      const theatreModeActive = this.defaultTheatreMode ? ' vjs-icon-theatre-active' : ''
+
+      const VjsButton = videojs.getComponent('Button')
+      const theatreModeButton = videojs.extend(VjsButton, {
```

Comment:
Just a renaming suggestion. Won't hurt if not taken.
```suggestion
      const toggleTheatreModeButton = videojs.extend(VjsButton, {
```

Reply:
No opinion, you are welcome to change this. I've never been a good bikeshedder.